pages/learn/manage/device-statuses.md

```diff
@@ -11,36 +11,69 @@ Device statuses are displayed on the Devices page and the Device Summary page. A
-| **Configuring**                              | The device is applying OS configuration.                                                                                                                                         |
-| **Updating**                                 | The device is updating to a new release.                                                                                                                                         |
+| **Operational**                              | The device is functioning normally and communicating with the cloud within the expected interval. In the standard configuration, this means both [cloudlink][cloudlink] and the device heartbeat API are working. If [cloudlink is disabled][bandwidth-reduction], the device will still appear `Operational` if the heartbeat API is functional.   |
+| **Reduced Functionality**<sup>1</sup>        | The device is connected to the backend but is experiencing limited functionality due to either **missing recent Heartbeat API communications** or **lack of connection to cloudlink**.                                                                                       |
+| **Disconnected**                                  | The device is not connected to cloudlink and has not communicated via the Heartbeat API for a duration exceeding the [API polling interval][poll-interval]. This status indicates that the device is completely offline and unable to send or receive any data from the cloud.                                                                                                                                                           |
+| **Configuring**                              | The device is applying OS configuration. For example, an OS upgrade.                                                                                                                                         |
+| **Updating**                                 | The device is updating to a new App release. In future versions this will also reflect OS updates.                                                                                                                                        |
 | **Post Provisioning**                        | The device has been [provisioned][device-provisioning] but has not yet come online.                                                                                              |
 | **Inactive**                                 | The device has been [deactivated][deactivated] or has been [preregistered][preregistered] but has not yet connected to the {{ $names.cloud.lower }} API.                         |
 | **Frozen**                                   | The device has been frozen because it's outside the organization's allowance, or is in a paid [fleet type][fleet type] on a free tier organization.                              |
 
+
 ## Device Connectivty states
```

Comment:
typo

```suggestion
## Device Connectivity states
```

Reply:
I think we need to have a CTA that links to the blog post on this page - as that details each of the direct path changes _e.g. online to operational._  Perhaps detailing the date this changed

e.g. [date] We have recently made changes to the naming of device statuses. You can find out more details here.